src/app/modules/customer-management/components/projects/components/services/project.service.ts

```diff
@@ -32,10 +33,17 @@ export class ProjectService {
 
   updateProject(projectData): Observable<any> {
     const { id } = projectData;
+    if (this.isDevelopment) {
+      if (projectData.status === 'active') {
+        projectData.status = 1;
+      }
+    }
```

Comment:
Maybe here you can write one if statement like this: if (this.isDevelopment && projectData.status === 'active')